### PR TITLE
chore: fix typos

### PR DESCRIPTION
**Pull request by kenji-miyake**
## Description

Fixed minor typos.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by kosuke55 — APPROVED** [review]
Thanks! LGTM

**Review by taikitanaka3 — APPROVED** [review]

**Review by rej55 — APPROVED** [review]
LGTM

File: control/trajectory_follower/include/trajectory_follower/mpc.hpp [hunk 1]
```diff
@@ -227,16 +227,16 @@ class TRAJECTORY_FOLLOWER_PUBLIC MPC
...
    */
   MPCMatrix generateMPCMatrix(
-    const trajectory_follower::MPCTrajectory & reference_trajectory, const float64_t predition_dt);
+    const trajectory_follower::MPCTrajectory & reference_trajectory, const float64_t prediction_dt);
   /**
    * @brief generate MPC matrix with trajectory and vehicle model
    * @param [in] mpc_matrix parameters matrix to use for optimization
    * @param [in] x0 initial state vector
-   * @param [in] precition_dt predition deleta time
+   * @param [in] precition_dt prediction delta time
```
Comment (shmpwk) [inline control/trajectory_follower/include/trajectory_follower/mpc.hpp:235]:
```suggestion
   * @param [in] prediction_dt prediction delta time
```

Comment (kenji-miyake) [reply control/trajectory_follower/include/trajectory_follower/mpc.hpp:235]: Thanks!

**Review by TakaHoribe — APPROVED** [review]
All items are for planning or control. Thank you so much.

**Review by mkuri — APPROVED** [review]
LGTM

**Comment by shmpwk** [comment]
Thank you! One more please  https://github.com/autowarefoundation/autoware.universe/actions/runs/3287712374/jobs/5417186032 :bow:

**Comment by kenji-miyake** [comment]
Sure!

**Comment by kenji-miyake** [comment]
I've used `calc` instead of `calculation` because `calculation` inserts a new line due to the line length.

**Review by shmpwk — APPROVED** [review]

**Review by satoshi-ota — APPROVED** [review]
LGTM :+1:

**Review by takayuki5168 — APPROVED** [review]

**Pull request merged**